src/Tizen.Multimedia/AudioManager/AudioStreamPolicy.cs

```diff
@@ -304,6 +306,76 @@ public void RemoveDeviceForStreamRouting(AudioDevice device)
                 ThrowIfError("Failed to remove device for stream routing");
         }
 
+        /// <summary>
+        /// Gets or sets the preferred input device.
+        /// </summary>
+        /// <value>
+        /// The <see cref="AudioDevice"/> instance.<br/>
+        /// The default is null which means any device is not set on this property.
+        /// </value>
+        /// <remarks>
+        /// This property is to set a specific built-in device when the system has multiple devices of the same built-in device type.
+        /// When there's only one device for a built-in device type in the system, nothing will happen even if this property is set successfully.
+        /// </remarks>
+        /// <exception cref="ArgumentException">A device is not for input.</exception>
+        /// <exception cref="InvalidOperationException">An internal error occurs.</exception>
+        /// <exception cref="AudioPolicyException">A device is not supported by this <see cref="AudioStreamPolicy"/> instance.</exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="AudioStreamPolicy"/> has already been disposed of.</exception>
+        /// <seealso cref="AudioManager.GetConnectedDevices()"/>
+        /// <since_tizen> 6 </since_tizen>
+        public AudioDevice PreferredInputDevice
+        {
+            get
+            {
+                Interop.AudioStreamPolicy.GetPreferredDevice(Handle, out var inDeviceId, out _).
```

Comment:
`GetPreferredDevice()` method is called but its return value is not used in `PreferredInputDevice`'s return value.
You can say `GetPreferredDevice()` is only for checking mismatch in native side, but I think it's not usual way.

How about creating instance `AudioDevice` based on its output?
or Rename `GetPreferredDevice()` to `ValidatePreferredDeviceId` or something like that?

Reply:
1. Creating instance of AudioDevice is not possible now. We can only get the instance from the device list.

2. The native getter API should take at least one out parameter otherwise it'll return invalid param error. Just renaming is not changing this situation. It might need one another internal native API without out parameter. Rather, I'd like to use this public getter API and add a log with the result of out parameter. How about this?

Reply:
Okay. And please add some additional information for pinvoke call, as we're discussed.

Reply:
The revision has been done.
@myroot Could you plz review again? Thanks in advance.